programming/sublime-text/pspec.xml

```diff
@@ -20,17 +22,39 @@
     </Source>
 
     <Package>
-        <Name>sublime-text-3</Name>
+        <Name>sublime-text</Name>
+        <Summary xml:lang="en">Sublime Text is a sophisticated text editor for code, markup and prose.</Summary>
+        <Description xml:lang="en"> Sublime Text is a sophisticated text editor for code, markup and prose.</Description>
+        <PartOf>programming</PartOf>
         <Files>
-            <Path fileType="*">/</Path>
+            <Path fileType="data">/opt/sublime_text</Path>
+            <Path fileType="data">/usr/bin/sublime_text</Path>
+            <Path fileType="data">/usr/share/applications/sublime_text.desktop</Path>
         </Files>
+        <AdditionalFiles>
+            <AdditionalFile owner="root" permission="0644" target="/usr/share/applications/sublime_text.desktop">sublime_text.desktop</AdditionalFile>
+        </AdditionalFiles>
         <RuntimeDependencies>
             <Dependency>libxscrnsaver</Dependency>
             <Dependency>gconf</Dependency>
         </RuntimeDependencies>
     </Package>
 
     <History>
+        <Update release="11">
```

Comment:
Should probably squash those releases into one for the initial PR.

Also, date format is MM-DD-YYYY here